include/revng/Pipeline/Context.h

```diff
@@ -62,6 +62,16 @@ class Context {
     return Globals.deserialize(GlobalName, Buffer);
   }
 
+  llvm::Expected<std::unique_ptr<Global>>
+  cloneGlobal(llvm::StringRef GlobalName) {
+    return Globals.clone(GlobalName);
+  }
+
+  llvm::Error applyDiffToGlobal(llvm::StringRef GlobalName,
+                                const llvm::MemoryBuffer &Buffer) {
```

Comment:
Because the diff deserialization is a specialization of the global, so it cannot be done here
(I'm also not sure that it would work if we decided that diff = tupletreediff, since there might be some erasure)

Reply:
Uhm, OK.